api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -120,7 +120,7 @@
       (is (:created workload) "should have a created timestamp")
       (is (:started workload) "should have a started timestamp")
       (is (= (:email @endpoints/userinfo) (:creator workload)) "creator inferred from auth token")
-      (let [include [:pipeline :executor :project]]
+      (let [include [:pipeline :project]]
```

Comment:
Maybe they were both `nil`?

Reply:
When I ran the tests, the request didn't have `executor` field while the response got all three fields with not nil values. Apparently:
```
(deftest ^:parallel test-exec-wgs-workload-specifying-cromwell
  ;; All modules make use of the same code/behavior here, no need to spam Cromwell
  (test-exec-workload (-> (workloads/wgs-workload-request (UUID/randomUUID))
                          (rename-keys {:executor :cromwell}))))
```
is not a right test as it renames the request key.

Reply:
Ran it again, here is the error:
```clojure
FAIL in (test-exec-wgs-workload-specifying-cromwell) (v1_endpoint_test.clj:132)
calling api/v1/exec with ExternalWholeGenomeReprocessing workload request
expected: (= (included request) (included workload))
  actual: (not (= {:pipeline "ExternalWholeGenomeReprocessing", :project "(Test) rex/GH-1130-purge-envs-clj"} {:executor "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org", :pipeline "ExternalWholeGenomeReprocessing", :project "(Test) rex/GH-1130-purge-envs-clj"}))
```